src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -108,6 +109,29 @@ public static JsonObject createStaffSlipContext(Request request) {
     return createStaffSlipContext(request.getItem(), request);
   }
 
+  public static JsonObject addPrimaryServicePointNameToStaffSlipContext(JsonObject entries, ServicePoint primaryServicePoint) {
+    if (primaryServicePoint == null) {
+      log.info("addPrimaryServicePointNameToStaffSlipContext:: parameters entries: {}, primaryServicePoint = null", entries);
```

Comment:
Please consider updating the message in the code to accurately reflect the executed condition